mockgcp/go.mod

```diff
@@ -1,159 +1,196 @@
 module github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp
 
-go 1.24
+go 1.24.0
 
 toolchain go1.24.5
 
+replace github.com/GoogleCloudPlatform/k8s-config-connector => ./..
```

Comment:
Is this aimed to sync the GCP lib between KCC and MockGCP? 
I see the benefits but it also feels hacky.

Reply:
What do you propose?

Reply:
If this is to sync the GCP libs, I thought we already have `go.work` and can use `go work sync`  (using the version in MockGCP or whatever is higher).

Reply:
I wasn't able to get that to work.  Can you try?